sAlgorithm/ListFind.js

```diff
@@ -12,4 +12,4 @@ export function ListFind(){
     }
    console.log("don't work");
 }
-ListFind()*/
\ No newline at end of file
+ListFind()
```

Comment:
You export the function, if someone wants to run it, it can be imported in the other files and called. You don't need to call it here directly., Remove this